### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: cakephpsphinx/themes/cakephp/layout.html [hunk 1]
```diff
@@ -21,7 +21,17 @@
...
-    <link href="{{ pathto('_static/favicon.png', 1) }}" type="image/png" rel="icon" />
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ pathto('_static/favicons/apple-touch-icon.png', 1) }}">
+    <link rel="icon" type="image/png" sizes="32x32" href="{{ pathto('_static/favicons/favicon-32x32.png', 1) }}">
+    <link rel="icon" type="image/png" sizes="16x16" href="{{ pathto('_static/favicons/favicon-16x16.png', 1) }}">
+    <link rel="manifest" href="{{ pathto('_static/favicons/manifest.json', 1) }}">
+    <link rel="manifest" href="{{ pathto('_static/favicons/favicon.ico', 1) }}">
+    <link rel="mask-icon" href="{{ pathto('_static/favicons/safari-pinned-tab.svg', 1) }}" color="#5bbad5">
+    <meta name="msapplication-config" content="{{ pathto('_static/favicons/browserconfig.xml', 1) }}" />
+    <meta name="apple-mobile-web-app-title" content="CakePHP">
+    <meta name="application-name" content="CakePHP">
```
Comment (markstory) [inline cakephpsphinx/themes/cakephp/layout.html:32]: Should this reflect the plugin or version that the site is for?

Comment (Arhell) [reply cakephpsphinx/themes/cakephp/layout.html:32]: maybe write what you need I will change

Comment (Arhell) [reply cakephpsphinx/themes/cakephp/layout.html]: i removed these settings

**Comment by othercorey** [comment]
@Arhell Can you open PR with ready to review?

**Pull request merged**